packages/schematics/integration/integration-test-bed/initialize-integration-test-bed.ts

```diff
@@ -0,0 +1,38 @@
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+import * as nixtFacade from '../nixt-facade';
+import { IntegrationTestBed } from './integration-test-bed';
+
+class IntegrationTestBedImpl {
```

Comment:
I thought of this more like a module internal implementation of the IntegrationTestBed interface

Reply:
An instance if `IntegrationTestBedImpl` is expose via the factory function below.
That's why it isn't internal anymore.

The file would become event more readable if it contains just one module instead of two.

Reply:
factory returns a Promise<IntegrationTestBed>, but yeah you're right, I'll merge it in the next refactoring PR :-)
I also wanted to extract some more helper Promises for setting up Angular-specific use-cases.